clamav-unofficial-sigs.sh

```diff
@@ -2504,23 +2528,30 @@ if [ "$sanesecurity_enabled" == "yes" ] ; then
         echo "$current_time" > "${work_dir_work_configs}/last-ss-update.txt"
         xshok_pretty_echo_and_log "Sanesecurity Database & GPG Signature File Updates" "="
         xshok_pretty_echo_and_log "Checking for Sanesecurity updates..."
-				# shellcheck disable=SC2086
-        sanesecurity_mirror_ips="$(dig $dig_proxy +ignore +short "$sanesecurity_url")"
-        # Add fallback to host if dig returns no records
+				if [ -z "$dig_bin" ] ; then
```

Comment:
This condition is inverted. It should be `if [ ! -z "$dig_bin" ] ; then` (if dig exists, then use it)